OMCompiler/Compiler/Template/CodegenFMU.tpl

```diff
@@ -1166,7 +1166,9 @@ match platform
   <%\t%>$(LD) -o <%modelNamePrefix%>$(DLLEXT) $(OFILES) $(RUNTIMEFILES) <%dirExtra%> <%libsPos1%> @BSTATIC@ <%libsPos2%> @BDYNAMIC@ $(LDFLAGS)
   <%\t%>cp <%fileNamePrefix%>$(DLLEXT) <%fileNamePrefix%>_FMU.libs ../binaries/$(FMIPLATFORM)/
   endif
+  if not Flags.getConfigEnum(Flags.FMI_FILTER) == Flags.FMI_BLACKBOX then
```

Comment:
This is not correct!
it should be 
<%if not Flags.getConfigEnum(Flags.FMI_FILTER) == Flags.FMI_BLACKBOX then
'<%\t%>head -n20 Makefile > ../resources/$(FMIPLATFORM).summary'
%>

Reply:
Thanks adrian, this is fixed with this https://github.com/OpenModelica/OpenModelica/pull/906/commits/1d88860fcd180400750ed5fc38dd6966d25b444a